src/server/pachyderm_test.go

```diff
@@ -10622,18 +10622,20 @@ func TestSpout(t *testing.T) {
 		iter, err := c.SubscribeCommit(pipeline, "master", nil, "", pfs.CommitState_FINISHED)
 		require.NoError(t, err)
 
-		for i := 0; i < 10; i++ {
+		var files []*pfs.FileInfo
+		for i := 0; len(files) < 1000; i++ {
```

Comment:
Previously, the loop was going through 10 times, now it goes for as many commits as necessary. What changed in the behavior? In both cases, we should be checking for 1000 files.

Reply:
I assumed you increased this just to make the test more stringent. Not sure what you mean by 'changed in the behavior'.